src/main/java/com/commercetools/sync/commons/utils/SyncSolutionInfo.java

```diff
@@ -12,19 +11,15 @@
      *
      * <p>A User-Agent header with a solution information looks like this:
      * {@code commercetools-jvm-sdk/1.4.1 (AHC/2.0) Java/1.8.0_92-b14 (Mac OS X; x86_64)
-     * {@code implementationTitle}/{@code implementationVersion}
-     * (+https://github.com/commercetools/commercetools-sync-java; +ps-dev@commercetools.com)}</p>
+     * {implementationTitle}/{implementationVersion}}</p>
      *
      */
     public SyncSolutionInfo() {
-        final String libraryName = "commercetools-sync-java";
         final String implementationTitle = ofNullable(getClass().getPackage().getImplementationTitle())
-            .orElse(libraryName);
+            .orElse("commercetools-sync-java");
         final String implementationVersion = ofNullable(getClass().getPackage().getImplementationVersion())
```

Comment:
As we saw before, this doesn't work, right? `getImplementationVersion` always returns `unspecifed`?

Reply:
you mean to check if it's unspecified? 
6950522

Reply:
you mean doesn't work in the tests? if you pack jar properly - it should work

Reply:
> wrong commit in the PR description?

No, it is correct this actually fixes it. Because the version variable is declared in `project-info.gradle` so it should be applied before `package.gradle`